docs/HowRNDesktopAppWorks.md

```diff
@@ -12,20 +11,20 @@ There are 3 participants in react-native-desktop application:
 - Bundler
 - JS server (right now called ubuntu-server, but this is subject to change)
 
-`Qt application` - built from a project in `desktop` folder. When runs it establishes communication with JS server and shows appropriate UI. 
+`Qt application` - built from a project in `desktop` folder. When runs it establishes communication with JS server and shows appropriate UI.
 
-`Bundler` - this server runs on developer's machine and provides access to js files from the project. Also, it can generate "bundle" - single js file with all project code in it. Not needed for distribution. 
+`Bundler` - this server runs on developer's machine and provides access to js files from the project. Also, it can generate "bundle" - single js file with all project code in it. Not needed for distribution.
```

Comment:
when bundler is not used for generating a single js file, is it's primary function to assist with hot reloading? or is it responsible for something else? (pardon my ignorance!)

Reply:
Correct, when hot reloading enabled, bundler modifies single js file.

Reply:
thanks!